src/RileyMaze/toolbox.js

```diff
@@ -66,6 +66,7 @@ var RMZToolbox = GObject.registerClass(class RMZToolbox extends Toolbox {
             new GLib.Variant('(sava{sv})', ['reset', [], {}]),
             null, Gio.DBusCallFlags.NONE, -1, null);
         this._unitTopic.reset();
+        this._levelTopic.discardChanges();
```

Comment:
Does this need an accompanying `this._instructionsTopic.discardChanges()`?

Reply:
Yeah I didn't have time to get to the bottom of this, but the instructionsTopic was already working with the reset button before this change so adding it here called it twice and added empty lines to the codeview.